src/antares/service/api_services/study_api.py

```diff
@@ -104,3 +105,11 @@ def delete(self, children: bool) -> None:
             self._wrapper.delete(url)
         except APIError as e:
             raise StudyDeletionError(self.study_id, e.message) from e
+
+    def create_variant(self, variant_name: str) -> str:
+        url = f"{self._base_url}/studies/{self.study_id}/variants?name={variant_name}"
+        try:
+            response = self._wrapper.post(url)
+            return response.text
```

Comment:
Adapt the code to return a Study.
You should do the same thing you did in the other create_variant function you introduced (but this time using read_areas())